test/AspNet.Security.OAuth.Providers.Tests/Strava/StravaTests.cs

```diff
@@ -18,6 +19,11 @@ public class StravaTests : OAuthTests<StravaAuthenticationOptions>
         public StravaTests(ITestOutputHelper outputHelper)
         {
             OutputHelper = outputHelper;
+
+            LoopbackRedirectHandler = new StravaLoopbackRedirectHandler()
```

Comment:
I've merged some refactoring into the dev branch to remove the need to create a custom loopback handler. If you rebase onto dev, this can then be simplified to just this and you can delete `StravaLoopbackRedirectHandler`:

```csharp
LoopbackRedirectHandler.LoopbackParameters.Add("scope", "read");
```